### DISCUSSION TITLE
在uos家庭版21.3.1上跑起来了

### DISCUSSION THREAD

**Discussion by xuanxuan1231** [Class-Widgets/Class-Widgets#207] [展示]
特 大 喜 报

![image](https://github.com/user-attachments/assets/45328109-1941-4c90-9c2c-fb852bf8bcc0)
然而窗口好像还没透明

**Comment by xuanxuan1231** [comment]
设置没问题 然而插件广场一开就退

**Comment by RinLit-233-shiroko** [comment]
啊这

**Comment by xuanxuan1231** [comment]
所有的浮窗（包括下拉栏）都会出现：
![image](https://github.com/user-attachments/assets/4d06e8bb-f38a-4d35-9ee1-8a244d4b9118)

**Reply by RinLit-233-shiroko** [reply]
![image](https://github.com/user-attachments/assets/7dc74548-d251-4793-9a88-da40af622398)
是不是没有开窗口特效，打开再看看有透明吗

**Reply by xuanxuan1231** [reply]
> 是不是没有开窗口特效，打开再看看有透明吗

我的问题
开了 有了

**Reply by RinLit-233-shiroko** [reply]
那就好）

**Reply by RinLit-233-shiroko** [reply]
那插件广场呢，可以找到原因吗

**Reply by xuanxuan1231** [reply]
> 那插件广场呢，可以找到原因吗

现在使用的版本是 #197 的，后面的没来及同步 

![image](https://github.com/user-attachments/assets/93f29a2a-2ce3-4b85-a03f-4b397092afd1)

**Reply by RinLit-233-shiroko** [reply]
ok

**Comment by pizeroLOL** [comment]
好家伙

**Comment by MKStoler4096** [comment]
有救了！有救了！
